### PR TITLE
minor functionality and fixes

### PR DESCRIPTION
**Pull request by alexwhan**
This is a collection of the pull requests we discussed previously.

### CODE REVIEW

File: R/groups.R [hunk 1]
```diff
@@ -180,7 +175,7 @@ clean_group <- function(id, ask = TRUE) {
         }
 
         # Delete child groups
-        child_groups <- rev(get_groups(groups = id))
+        child_groups <- get_groups(groups = id)
```
Comment (byzheng) [inline R/groups.R]: Do you test here? I believe I have to use function `rev` to delete groups.

File: R/groups.R [hunk 2]
```diff
@@ -154,24 +154,19 @@ clean_group <- function(id, ask = TRUE) {
     }
 
     recursive_clean_group <- function(id) {
-        # Delete platforms
```
Comment (byzheng) [inline R/groups.R]: I believe I have to delete platfrom before deleting streams.  Need to test here.

File: R/groups.R (hunk 1 above)
Comment (alexwhan) [reply R/groups.R]: This file got caught up in the merge inadvertently, it's removed now

File: R/groups.R (hunk 2 above)
Comment (alexwhan) [reply R/groups.R]: This file got caught up in the merge inadvertently, it's removed now

**Comment by alexwhan** [comment]
Good pickup @byzheng, that was because I didn't have your changes on master when i started the branch. I've checked out your changes and commited on dev, so those two files shouldn't change in the pull request now

**Comment by byzheng** [comment]
It should be fine now. Can merge it if you like.

**Pull request merged**